packages/kubernetes-1.19/kubelet-config

```diff
@@ -93,6 +93,12 @@ topologyManagerPolicy: {{settings.kubernetes.topology-manager-policy}}
 {{#if settings.kubernetes.pod-pids-limit includeZero=true}}
 podPidsLimit: {{settings.kubernetes.pod-pids-limit}}
 {{/if}}
+{{#if settings.kubernetes.image-gc-high-threshold-percent includeZero=true}}
+imageGCHighThresholdPercent: {{default 85 settings.kubernetes.image-gc-high-threshold-percent}}
+{{/if}}
+{{#if settings.kubernetes.image-gc-low-threshold-percent includeZero=true}}
+imageGCLowThresholdPercent: {{default 80 settings.kubernetes.image-gc-low-threshold-percent}}
+{{/if}}
```

Comment:
I'll choose keeping the `{{#if}}` blocks to stick to convention and clarity -- also removing hardcoded default values (in case kubelet changes from 85/80 high/low)